frontend/containers/AuctionsFakeData.vue

```diff
@@ -120,6 +160,8 @@ export default Vue.extend({
                     this.selectedAuction.isFinished = true;
                     this.selectedAuction.endDate = new Date();
                     this.selectedAuction.transactionAddress = faker.finance.ethereumAddress();
+                    this.walletVatDai = this.walletVatDai.minus(this.transactionAmountDai!!);
```

Comment:
What's `!!` suppose to do here?

Reply:
Well, the second one was accidental 😅
One is to remove `undefined` since this step can't really be reached with `transactionAmountDai` being `undefined` and `minus` doesn't work with `BigNumber | undefined`